test/extended/extended_test.go

```diff
@@ -0,0 +1,70 @@
+package extended
+
+import (
+	"flag"
```

Comment:
This is copying the E2E framework.

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/test/e2e/e2e_test.go#L20